elasticsearch/utils/logging

```diff
@@ -161,52 +113,6 @@ wait_for_port_open() {
     exit 1
 }
 
-build_jks_from_pem() {
```

Comment:
we're still going to need to keep some of this because we're converting the `pem` files into `pkcs12` in here. we just don't need the latter half where we convert `pkcs12` into `jks`

Reply:
thanks for clarifying. I've restored the functions and removed out the jks portions.